test/fuzzing/build.sh

```diff
@@ -0,0 +1,28 @@
+export FUZZER_DIR=$SRC/linkerd2/test/fuzzing
+
+mkdir $SRC/linkerd2/test/fuzzing
+
+mv $FUZZER_DIR/inject_fuzzer.go $SRC/linkerd2/pkg/inject/
```

Comment:
Same thing with `$FUZZER_DIR`, of course.

(Since the CI shellcheck https://github.com/linkerd/linkerd2/blob/main/.github/workflows/static_checks.yml#L60 calls shellcheck-all which checks only files with a `/usr/bin/env (sh|bash)` shebang https://github.com/linkerd/linkerd2/blob/main/bin/shellcheck-all#L13 this file currently goes under the shellcheck radar)